examples/test/qlib/SqlUtil/PgsqlSqlUtil.qtest

```diff
@@ -166,7 +177,13 @@ class PgsqlTest inherits SqlTestBase {
 
         if (ds) {
             foreach hash h in (CustomColumns.pairIterator()) {
-                any val = h.value.typeCode() == NT_HASH ? h.value.value : h.value;
+                any val;
```

Comment:
this would be more elegantly written as
any val = h.value bind ?? h.value.value ?? h.value;